src/nvim/quickfix.c

```diff
@@ -80,13 +80,22 @@ struct qfline_S {
 #define LISTCOUNT   10
 #define INVALID_QFIDX (-1)
 
+// Quickfix list type.
+typedef enum
+{
+  QFLT_QUICKFIX,  // Quickfix list - global list
```

Comment:
When commenting members of a struct/union/enum inline, Doxygen-style comments should be used, that is `///<`. See http://doxygen.nl/manual/docblocks.html#memberdoc